tests/reporter/kubernetes.go

```diff
@@ -240,8 +240,22 @@ func (r *KubernetesReporter) logDMESG(virtCli kubecli.KubevirtClient, logsdir st
 				fmt.Fprintf(os.Stderr, "failed to get virt-handler pod on node %s: %v\n", node, err)
 				return
 			}
+
+			commands := []string{
+				virt_chroot.GetChrootBinaryPath(),
+				"--mount",
+				virt_chroot.GetChrootMountNamespace(),
+				"exec",
+				"--",
+				"/proc/1/root/bin/dmesg",
```

Comment:
You can probably use `/usr/bin/dmesg` here since you now switch the namespace, but this will work too.

Reply:
whichever you prefer?

Reply:
Just curiosity, why do we use `virt-chroot --mount` and not `nsenter -t -1 -m`? What is the extra value of `virt-chroot`? Just not depending on the `nsenter` binary of the container or something else?

Reply:
Mostly because it subsums a few features into one binary:

 * changing the namespace
 * dropping capabilities
 * setting process limits

We could "save" the namespace switches from this binary, but it is trivial to do and we would have a more complicated call-model:

 * call nsenter
 * call after nsenter our golang binary to drop privileges, set limits ...
 
 We can't drip privileges and can't set limits before doing the namespace switch.
 
 @AlonaKaplan let me know if that does not sound convincing.

Reply:
It sounds convincing as long as we use the described options. But here we just switch the mount namespace, don't we?

Reply:
Well the binary has the feature of switching the namespace already. Therefore I would prefer to not add another dependency.

Reply:
If there is a consensus , please take another look, since the lgtm was removed when i updated code, thanks